kv/kv.go

```diff
@@ -46,9 +46,9 @@ const UnCommitIndexKVFlag byte = '1'
 // Those limits is enforced to make sure the transaction can be well handled by TiKV.
 var (
 	// TxnEntrySizeLimit is limit of single entry size (len(key) + len(value)).
-	TxnEntrySizeLimit uint64 = config.DefTxnEntrySizeLimit
+	TxnEntrySizeLimit = atomic.NewUint64(10485760) //DefTiDBTxnEntrySizeLimit
```

Comment:
Why not using `DefTiDBTxnEntrySizeLimit` directly? BTW `10485760` is not equal to `DefTiDBTxnEntrySizeLimit((6 * 1024 * 1024))`

Reply:
Afaik this is due to circular dependencies since `sessionctx/variable` imports `kv`. But I agree we can change the starting value to 6M. It will be overwritten in the startup procedure though, as the sysvar cache is populated.